### PR TITLE
Cleanup for "Developing smart contracts" on the IC and "multi-actors tutorial"

### PR DESCRIPTION
**Pull request by andreacerulli**
Making the terminology language more accurate in the pages "Developing smart contracts" and "Multi-actors tuorial".

### CODE REVIEW

File: modules/developers-guide/pages/sdk-guide.adoc [hunk 1]
```diff
@@ -6,27 +6,22 @@
...
+* *Local development:* you start a local canister execution environment simulating the {IC} blockchain on your computer. Then you write, compile, install and iteratively update your canisters in the local execution environment. This gives you the possibility to test your canisters locally without the need to use cycles to power them.
 
-If you prefer to do all of your development work without connecting to a local or remote network, you can follow the second path and only connect to the network when you are ready to compile and deploy.
-With this option, you only connect to the Internet Computer network and register for a unique identifier after you have a program ready to deploy.
+* *On-chain deployment:* once your dapp is ready you can then deploy it to the {IC} blockchain mainnet, making it available for the world to use it. Note that your canisters need have cycles to be able to run on the {IC} blockchain mainnet, as discussed in link:../developers-guide/concepts/tokens-cycles{outfilesuffix}[Tokens and cycles].
```
Comment (yvonneanne) [inline modules/developers-guide/pages/sdk-guide.adoc:25]:
```suggestion
* *On-chain deployment:* once your dapp is ready you can then deploy it to the {IC} blockchain mainnet, making it available for the world to use it. Note that your canisters need have cycles to be able to run on the {IC} blockchain mainnet, as discussed in link:../developers-guide/concepts/tokens-cycles{outfilesuffix}[Tokens and cycles].
```
```suggestion
* *On-chain deployment:* once your dapp is ready you can then deploy it to the {IC} blockchain mainnet, making it available for the world to use it. Note that your canisters need to have cycles to be able to run on the {IC} blockchain mainnet, as discussed in link:../developers-guide/concepts/tokens-cycles{outfilesuffix}[Tokens and cycles].
```

File: modules/developers-guide/pages/sdk-guide.adoc [hunk 2]
```diff
@@ -6,27 +6,22 @@
...
 ifdef::env-github,env-browser[:outfilesuffix:.adoc]
 
-The {platform} is poised to help you start a revolution with a new way to design, build, and release software.
+The {IC} blockchain is poised to help you start a revolution with a new way to design, build, and release dapps.
 
-== Developing open internet applications
+== Developing dapps
 
-Conceptually, the Internet Computer is an extension of the existing internet that enables a global network of computers to provide the computing power for running application software.
+The {IC} is a blockchain that runs canister smart contracts, which are code units bundling together WebAssembly bytecode and the memory pages the bytecode runs in. The {IC} is composed by individual subnet blockchains running in parallel and connected together by the use of Chain Key Cryptography. This means that canisters running on a subnet can seamlessly call canisters hosted in any other subnet of the {IC} blockchain. Moreover, the governance system of the {IC} can dynamically increase the capacity of the {IC} by adding new subnets, allowing dapps to scale out.
```
Comment (Dfinity-Bjoern) [inline modules/developers-guide/pages/sdk-guide.adoc:13]: Overall looks good and concise, but I think composed by -> composed of ?

**Review by Dfinity-Bjoern — APPROVED** [review]
Really good work, I like how your changes are very consistent and precise in their use of terminology and shorthands.

**Review by yvonneanne — APPROVED** [review]
Very nice!

File: modules/developers-guide/pages/sdk-guide.adoc (hunk 1 above)
Comment (andreacerulli) [reply modules/developers-guide/pages/sdk-guide.adoc]: fixed, thanks!

**Pull request merged**